deliver/lib/deliver/setup.rb

```diff
@@ -70,12 +70,16 @@ def generate_metadata_files(v, path)
 
       # get App icon + watch icon
       if v.large_app_icon.asset_token
-        app_icon_path = File.join(path, "app_icon.png")
+        app_icon_extension = "jpg" if v.large_app_icon.url.end_with? "jpg"
```

Comment:
according to apple documentation only jpg and png are supported so this should never happen, any ideas on further improving it?